.github/workflows/test-go.yml

```diff
@@ -1,12 +1,20 @@
 on:
   workflow_call:
     inputs:
+      name:
+        description: A suffix to be added to the matrix job names.
+        required: true
+        type: string
       go-arch:
         description: The execution architecture (arm, amd64, etc.)
         required: true
         type: string
+      extra-flags:
+        description: A space-separated list of additional build flags.
+        required: false
+        type: string
       extra-tags:
-        description: A comma-separated list of additional build tags.
+        description: A space-separated list of additional build tags.
```

Comment:
From [Go 1.13 release notes](https://golang.org/doc/go1.13#go-command):

The go build flag -tags now takes a comma-separated list of build tags, to allow for multiple tags in GOFLAGS. The space-separated form is deprecated but still recognized and will be maintained.